Dockerfile

```diff
@@ -1,8 +1,17 @@
-FROM yunspace/alpine-golang
+FROM golang:alpine
 
-ADD build/codeclimate-gofmt /usr/src/app/
+MAINTAINER hello@codeclimate.com
 
 RUN adduser -u 9000 -D app
+
+WORKDIR /code
+
+COPY build/ /usr/src/app
+COPY build/codeclimate-gofmt /usr/src/app/build/codeclimate-gofmt
+RUN chown -R app:app /usr/src/app
```

Comment:
Discussed in slack, but I'm not sure this is needed. Does the engine work without it?